Dockerfile.dapper

```diff
@@ -80,6 +80,8 @@ RUN curl -sLf ${!DOCKER_URL} -o /usr/bin/docker && chmod +x /usr/bin/docker
 ENV YQ_URL=https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_${ARCH}
 RUN curl -sLf ${YQ_URL} -o /usr/bin/yq && chmod +x /usr/bin/yq
 
+RUN zypper install -y jq
```

Comment:
why not add this to the install on line 18, we should at least put it higher up as it is less likely to change than the other dependencies.

Reply:
Ok, I'll move it. I just put it there because is right after where we install yq.